src/components/urlImport/index.js

```diff
@@ -247,42 +234,69 @@ const URLImportDialog = (props) => {
             aria-labelledby={buildID(ids.BASE_ID, ids.TITLE)}
         >
             <DialogTitle id={buildID(ids.BASE_ID, ids.TITLE)}>
-                {fmt(intl, "title")}
+                {t("title")}
                 <IconButton
                     aria-label="close"
                     className={classes.closeDialog}
-                    onClick={onClose}
+                    onClick={() => {
+                        setSubmittedMsg("");
+                        setErrorMsg("");
+                        onClose();
+                    }}
                 >
                     <CloseIcon />
                 </IconButton>
             </DialogTitle>
 
             <DialogContent>
                 <Typography className={classes.instructions}>
-                    {fmt(intl, "instructions")}
+                    {t("instructions")}
                 </Typography>
+
                 <URLImportTextField
-                    intl={intl}
                     path={path}
-                    showErrorAnnouncer={showErrorAnnouncer}
+                    successCallback={(msg) => setSubmittedMsg(msg)}
+                    errorCallback={(msg) => setErrorMsg(msg)}
```

Comment:
For some reason, calling these `successCallback` and `errorCallback` confused me.  Might just be me though.